test/integration/guest_views_user_profile_test.rb

```diff
@@ -1,10 +1,22 @@
 require "test_helper"
 
 class GuestViewsUserProfileTest < ActionDispatch::IntegrationTest
+  def login_store_admin
+    Store.first.update(approved: true)
+    User.find_by(username: "Dave").roles.create(name: 'store_admin')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.